### PR TITLE
#166698914 Fix - AssignAdmin role page does not update real-time

### PR DESCRIPTION
**Pull request by AmosWels**
#### What does this PR do?
- Fixes - `AssignAdmin role` page which does not update in real-time
- Clears the text box after the submission of the email is done

#### Description of Task to be completed?
- Currently, the AssignAdmin role page does not update real-time when the admin role has been assigned successfully to a new user. And Also the text in the input box is cleared.

#### How should this be manually tested?
- Log into the application.
- Access the Admin Panel, navigate to the Users panel and assign a new admin role.

#### What are the relevant pivotal tracker stories?
- [#166698914](https://www.pivotaltracker.com/story/show/166698914)

#### Background Context
- There is a need to improve UX when one assigns the admin role to a user. Currently, the AssignAdmin role page does not update real-time when the admin role has been assigned successfully to a new user. And Also the text in the input box is cleared.
#### Screenshots (If Applicable)
N/A

### CODE REVIEW

**Comment by sulenchy** [comment]
@AmosWels Pls add one more reviewer to your PR. Going forward, pls always assign at least three reviewers to your PR.

File: client/src/components/Admin/Users/AdminUsers.jsx [hunk 1]
```diff
@@ -34,21 +38,43 @@ export class Users extends Component {
    */
   handleSubmit = (event) => {
     event.preventDefault();
+    const { emailAddress } = this.state;
+    const emailData = emailAddress.trim();
     const userData = {
-      emailAddress: event.target.elements.userEmail.value,
+      emailAddress: emailData,
```
Comment (sulenchy) [inline client/src/components/Admin/Users/AdminUsers.jsx]: @AmosWels Good job here

**Comment by AmosWels** [comment]
> Good Job so far. I can see that the email input box refreshes after every click on the `assign admin role` button but I noticed the newly assigned users are not shown in the list of admin users. Pls review your implementation.

>The newly assigned admin shows at the bottom of the list. However, i noted to the backend that the name of the new admin is not returned at the point, only the email is returned. It's only returned after a refresh. They mentioned that they would fix it.

**Comment by AmosWels** [comment]
> Kindly take a look at the codeclimate issues. I think we can resolve the duplication issues faced here. Otherwise, good work!!

> I have resolved the code climate issues, kindly have a look at them @SoniaRMK

**Pull request merged**